express-routes/server.ts

```diff
@@ -0,0 +1,27 @@
+import express from 'express';
+
+const app = express();
+
+app.use((req, res, next) => {
+  console.log(new Date(), req.method, req.path);
+  next();
+});
+
+app.get('/', (req, res, next) => {
+  res.send('bananas');
+  next();
```

Comment:
You usually won't call `next()` if you send a response. You usually only call `next()` if you do not do anything with the request.

It doesn't hurt, though, and there are rare use cases where you would want to do it. But I would suggest you not do it in code you write at LFZ since it is not needed in anything we do.